linkedlist/main.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
instead of using main file i think it's better to use test file for testing this code [https://go.dev/doc/tutorial/add-a-test](you can read this for references)

Reply:
I've thought about that, but is this necessary? There isn't any needed type checks since there is no interface used, also most of the functions doesn't have a return value. Code will needs to get doubled to make all tests, and outputs won't be pretty